modules/gatsby/src/gatsby-node/process-nodes-markdown.js

```diff
@@ -1,37 +1,37 @@
 const path = require('path');
-const moment =require('moment');
+const moment = require('moment');
 // TODO/ib - remove
-const _ =require('lodash');
+const _ = require('lodash');
+const { log, COLOR } = require('../utils/log');
 
 module.exports.cleanupMarkdownNode = function cleanupMarkdownNode(
   { node, getNode, loadNodeContent, actions, createNodeId, reporter },
   pluginOptions
 ) {
-
   let processed = false;
 
   if (!processed) {
     switch (node.internal.mediaType) {
-    case `text/markdown`:
-    case `text/x-markdown`:
-      addSourceInstanceName(...arguments);
-      processed = true;
-      break;
+      case `text/markdown`:
```

Comment:
I have said this before, but I think we need to get your new editor setup so it reads formatting rules from files in the repo, better avoid unrelated formatting changes. Time to implement prettier.

Reply:
i'm all about settling for common prettier rules. we have a .eslintrc.json file in the repo but no .prettierrc or equivalent

Reply:
In the same repo we have a module ocular-dev-tools, I believe it includes prettier in its scripts, we should update.